install.sh

```diff
@@ -1038,11 +795,13 @@ RUBY_AUTH_LIB_FILES="src/authm_mad/remotes/ssh/ssh_auth.rb \
 # and remotes directory
 #-----------------------------------------------------------------------------
 
+REMOTE_FILES="src/vmm_mad/remotes/kvm/vgpu"
+
 MAD_SH_LIB_FILES="src/mad/sh/scripts_common.sh \
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

MAD_SH_LIB_FILES appears unused. Verify use (or export if used externally).